tlb/transactions.go

```diff
@@ -440,9 +437,6 @@ func (tx Transaction) IsSuccess() bool {
 		return true
 	case "TransTickTock":
 		t := tx.Description.TransTickTock
-		if t.Aborted {
```

Comment:
why this aborted tx is mean sucssed tx? i am learning ton, cloud you show me a doc for this? thx :)

Reply:
@mr-tron  @aleksej-paschenko